app/Http/Controllers/ApplicationByJobController.php

```diff
@@ -91,15 +93,19 @@ public function editBasics(JobPoster $jobPoster)
         $this->authorize('view', $application);
         $this->authorize('update', $application);
 
+        $viewTemplate = $jobPoster->isInStrategicResponseDepartment()
```

Comment:
I probably would have done something like
```php
$templatePath = $jobPoster->isInStrategicResponseDepartment() ? "/strategic_response_application" : "";
...
return view (
    "applicant$templatePath/application_post_01",
```
Just to avoid checking/possible typos on every render, but this works fine as-is.